pandas/parser.pyx

```diff
@@ -662,7 +662,11 @@ cdef class TextReader:
 
         if isinstance(source, basestring):
             if not isinstance(source, bytes):
-                source = source.encode(sys.getfilesystemencoding() or 'utf-8')
+                if compat.PY36 and compat.is_platform_windows():
+                    fs_encoding = 'mbcs'
```

Comment:
this also need to go in pandas/io/common in _get_filehandle_or_buffer so prob needs to be in a common routine